src/stars.coffee

```diff
@@ -60,8 +61,24 @@ class Stars extends Command
   installPackages: (packages, callback) ->
     return callback() if packages.length is 0
 
-    commandArgs = packages.map ({name}) -> name
-    new Install().run({commandArgs, callback})
+    toName = ({name}) -> name
+
+
+    list = new List()
+    listOptions =
+      argv:
+        dev: true
+        bare: true
+
+    list.listDevPackages listOptions, (error, devPackages) ->
```

Comment:
Can you share some thoughts on why both user and dev packages should be checked? I _think_ I would have expected only user packages to be checked, because that's how these packages will be installed.